src/lib/chat.ts

```diff
@@ -83,6 +83,19 @@ export interface WorkflowContent {
   projectId: string;
 }
 
+export interface PullRequestContent {
+  branchName: string;
+  commitMessage: string;
+  repositories: {
```

Comment:
why object with {url, owner, repo}... just make one flat object for PullRequestContent {..., repoUrl, repoOwner, repoName. ...}

PR looks good to me tho!